src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -68,6 +73,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  // Validates if an URL posted is valid
+  public static boolean urlValidator(String url) {
+
+    try {
+      new URL(url).toURI();
+      return true;
+    } catch (URISyntaxException exception) {
+      return false;
+    } catch (MalformedURLException exception) {
+      return false;
+    }
+  }
+
```

Comment:
This is neat! I'm wondering (because I don't know error handling very well yet) if maybe we should also log these errors to console, and say which one it is? I'm not sure I know the difference and it may reduce pain down the line when we're debugging why something isn't working like we expect

Reply:
Well the error is in case the URL isn´t valid but I didn´t add an action in case the program think is a URL but it is not, and I can´t let the log for the error because there shouldn´t be any at the pull request

Reply:
I'm not completely sure I understand - we've had error logging before (I think we currently log errors if someone tries to go to a page that doesn't exist), I think that all that we've had to remove is debugging stuff (say, printing the user every time we go to the user page). Definitely your call though :D

Reply:
Oh ok I get it, well I was thinking that at the end of the project try to break in all the possible ways the page so we can have all the errors that a user can have and then just explain to the user with a JS message